store-implementation/parquet-store/src/main/java/uk/gov/gchq/gaffer/parquetstore/utils/Visibilities/ArrayByteSequence.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.parquetstore.utils.visibilities;
```

Comment:
The folder structure needs to represent the package structure, visibilities is capitalised in the folder structure.